### ISSUE TITLE
In Hexen 2 I get stuck in carpets or pieces of geometry that seem flat

### ISSUE THREAD

**Issue by lucascalsilva** [fte-team/fteqw#294]
This happened quite a lot in the shops in Blackmarsh, where there are many carpets.

I have to jump through it to pass, but it gets annoying at some point.

**Comment by evilman666** [comment]
Yes I have that same issue using the latest GoG data files; the character also gets stuck at the end of hallways as well as carpets.
fteqw version 1.7.0.

**Comment by evilman666** [comment]
Seems to be a regression; if I use version 6202 from fte.triptohell.info, the issue is not there.

**Comment by Xylemon** [comment]
Can you provide a screenshot of the location and the name of the map for testing?
